install.ps1

```diff
@@ -0,0 +1,80 @@
+# Installs KoreBuild into a repo, overwriting all previous build scripts
+# This script is downloaded and executed without any of the corresponding scripts in this repo, so don't assume anything in this repo is available!
+
+param(
+    [string]$KoreBuildRepo,
+    [string]$KoreBuildBranch = "anurse/msbuild") # TEMPORARY until this is merged to dev
+
+# Prerequisites
+if(!(Get-Command git -ErrorAction SilentlyContinue)) {
+    throw "KoreBuild requires Git"
```

Comment:
> we're already using something that works well.

Ay there's the rub. We're not using git yet to synchronize korebuild version. We don't know all issues we'll hit once we start using it. But here are some things to consider

 - `git clean -xfd` doesn't delete sub-git folders. People have to re-train muscle memory to add the second "f" to actually clean everything `git clean -xffd`
 - `git pull` isn't enough. Homebrew has [extensive code](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cmd/update.sh) around `git pull` to ensure that if a simple pull fails, it can recover. We'll need something similar otherwise the standard fix will rename "just delete `.build`".
 - Switching branches. git checkout will not always work if there will be file conflicts as a result of the branch change.

I'm not opposed to using git per se ... but I assert there are simpler cache invalidation strategies.